src/Tribe/Attendees.php

```diff
@@ -553,6 +553,49 @@ private function generate_filtered_list( $event_id ) {
 		return array_filter( $rows );
 	}
 
+	/**
+	 * Sanitize rows for CSV usage.
+	 *
+	 * @since 4.10.7.2
+	 *
+	 * @param array $rows Rows to be sanitized.
+	 *
+	 * @return array Sanitized rows.
+	 */
+	public function sanitize_csv_rows( array $rows ) {
+		foreach ( $rows as &$row ) {
+			$row = array_map( [ $this, 'sanitize_csv_value' ], $row );
+		}
+
+		return $rows;
+	}
+
+	/**
+	 * Sanitize a value for CSV usage.
+	 *
+	 * @since 4.10.7.2
+	 *
+	 * @param mixed $value Value to be sanitized.
+	 *
+	 * @return string Sanitized value.
+	 */
+	public function sanitize_csv_value( $value ) {
```

Comment:
Hey @sc0ttkclark, this looks good at first glance. Looks like what you are doing is checking for the injection char, and if that character is the fist part of a rows string, you remove it, then recursively hit the row again to check the next char until it passes the test. After a little review, this looks like an effective solution. Possible to throw a few unit tests at this to confirm?

Reply:
Yep, doing that next.

I just changed the functionality to prefix instead of remove, it looks like it's the recommended solution from Zach and I's research.